dashboard/app/helpers/plc/evaluations_helper.rb

```diff
@@ -0,0 +1,34 @@
+module Plc::EvaluationsHelper
+  def determine_preferred_learning_modules level_source, script
```

Comment:
most of the actual logic for determining what the next thing to go to should be in `ScriptLevel#next_unpassed_progression_level` -- the stuff in ScriptLevelsHelper seems pretty thin and is mostly generating urls based on what `next_unpassed_progression_level` returns.

How would this look as an instance method on one of the arguments of this method (I would probably suggest trying to put it on Script first if level_source can be nil)?